Packs/Use_Case_Builder/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,4 @@
+
+#### Incident Fields
+
+- **Use Case Development Stage**
```

Comment:
Please replace this with a proper release notes message saying that the incident field is now searchable.

Reply:
done